analysis/src/SharedTypes.ml

```diff
@@ -425,6 +425,12 @@ module Completable = struct
     | CPObj of contextPath * string
     | CPPipe of contextPath * string
 
+  type typedContext = NamedArg of string
+
+  let typedContextToString typedContext =
+    match typedContext with
+    | NamedArg argName -> "NamedArg(" ^ argName ^ ")"
```

Comment:
This was a very quick solution just to complete the actual items. I understand this is probably not the right way to do it.

Reply:
I think this is just debug printing. Unless the comment refers to the type definition, which is perfectly fine for now.

Reply:
Yes, it was referring to the type def.